src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationemployment/api/integration/util/TestData.kt

```diff
@@ -0,0 +1,345 @@
+        booleanTrue,
+      ),
+    )
+
+    val updatedReadinessProfile_accpeted_1 = Optional.of(
+      ReadinessProfile(
+        newOffenderId,
+        updatedBookingId,
+        updatedBy,
+        modifiedTime,
+        updatedBy,
+        modifiedTime,
+        "1.0",
+        CapturedSpringConfigValues.objectMapper.readTree(CapturedSpringConfigValues.objectMapper.writeValueAsString(profile_accpeted_modified)),
+        CapturedSpringConfigValues.objectMapper.readTree("[]"),
+        booleanTrue,
+      ),
+    )
+    val readinessProfile_declined_1 = Optional.of(
+      ReadinessProfile(
+        newOffenderId,
+        newBookingId,
+        createdBy,
+        modifiedTime,
+        createdBy,
+        modifiedTime,
+        "1.0",
+        CapturedSpringConfigValues.objectMapper.readTree(CapturedSpringConfigValues.objectMapper.writeValueAsString(profile_declined)),
+        CapturedSpringConfigValues.objectMapper.readTree("{}"),
+        booleanTrue,
+      ),
+    )
+    val readinessProfile_declined_1_declined_list = Optional.of(
+      ReadinessProfile(
+        newOffenderId,
+        newBookingId,
+        createdBy,
+        modifiedTime,
+        createdBy,
+        modifiedTime,
+        "1.0",
+        CapturedSpringConfigValues.objectMapper.readTree(CapturedSpringConfigValues.objectMapper.writeValueAsString(profile_declined_declined_list)),
+        CapturedSpringConfigValues.objectMapper.readTree("{}"),
+        booleanTrue,
+      ),
+    )
+    val updatedReadinessProfile_declined_1 = Optional.of(
+      ReadinessProfile(
+        newOffenderId,
+        updatedBookingId,
+        updatedBy,
+        modifiedTime,
+        updatedBy,
+        modifiedTime,
+        "1.0",
+        CapturedSpringConfigValues.objectMapper.readTree(CapturedSpringConfigValues.objectMapper.writeValueAsString(profile_declinedModified)),
+        CapturedSpringConfigValues.objectMapper.readTree("[]"),
+        booleanTrue,
+      ),
+    )
+    val profile_IncorrectStatus: Profile = Profile(
+      profileStatus_SUPPORT_NEEDED, false, null, "prison1", StatusChange.NEW, mutableListOf(supportDeclined),
+      mutableListOf(
+        supportAccepted,
+      ),
+      supportDeclined, null,
+    )
+    val profile_NEW_BOTHSTATE_INCOORECT: Profile = Profile(
+      profileStatus_NO_RIGHT_TO_WORK, false, null, "prison2", StatusChange.NEW,
+      mutableListOf(
+        supportDeclined,
+      ),
+      mutableListOf(supportAccepted), supportDeclined, supportAccepted,
+    )
+
+    val readinessProfile = ReadinessProfile(
+      newOffenderId,
+      newBookingId,
+      createdBy,
+      modifiedTime,
+      createdBy,
+      modifiedTime,
+      "1.0",
+      CapturedSpringConfigValues.objectMapper.readTree(CapturedSpringConfigValues.objectMapper.writeValueAsString(profile)),
+      CapturedSpringConfigValues.objectMapper.readTree("{}"),
+      booleanTrue,
+    )
+
+    val updatedReadinessProfile = Optional.of(
```

Comment:
not necessary, in kotlin you wouldn't use Optional. 
Would just use nullable values and things like `?.let{}` to do `map`/`flatmap`